app/components/ui/Banner.tsx

```diff
@@ -1,23 +1,17 @@
 import React from 'react';
+import { icon } from 'assets';
 import styles from './Banner.module.scss';
 
 export const Banner = () => (
   <div className={styles.bannerContainer}>
-    <strong>COVID-19 VACCINE SITES: </strong>
-    Find out where to get a vaccine if you&apos;re eligible for a vaccine&#44; find a
-    {' '}
-    <a target="_blank" rel="noreferrer" href="https://sf.gov/vaccine-sites">site near you</a>
-    .
-    <br />
-    <strong>Need help applying for rent payment assistance?</strong>
-    {' '}
-    Check out
-    {' '}
-    <a target="_blank" rel="noreferrer" href="https://docs.google.com/document/d/1t-RM30QQHA5UQdzIGHrD6kaFoBMrwmCBM7mEOBD8a9U/edit">Code Tenderloin&apos;s workshops</a>
-    {' '}
-    and
-    {' '}
-    <a target="_blank" rel="noreferrer" href="https://www.homeownershipsf.org/application-assistance-for-renters/">HomeownershipSF&apos;s resources</a>
-    .
+    <div className={styles.bannerContent}>
```

Comment:
What's the purpose of this inner div? The CSS style doesn't exist, either. If it's not needed, can we remove this layer and have the rest of the content directly under the `bannerContainer`?

Reply:
I ended up using it to align the content with the homepage tiles per Dongyeon's designs. Currently we set our tile layout by setting a percentage width and a max-width. I just copied those properties onto the banner for now, rather than create my own kind of grid layout that would align with the homepage. I think I/we could go back and fix it up better, but I didn't want to sink too much more time into it.